### PR TITLE
Template for KV explorer with Labels

### PR DESCRIPTION
**Pull request by kate-goldenring**
Template for changes in https://github.com/fermyon/spin-kv-explorer/pull/27

Should be merged after

Separated so release version and hash can be updated to the appropriate release number with changes from #27

### CODE REVIEW

File: readme.md
Comment (karthik2804) [inline readme.md]:
```suggestion
source = { url = "https://github.com/fermyon/spin-kv-explorer/releases/download/v0.10.0/spin-kv-explorer.wasm", digest = "sha256:07f5f0b8514c14ae5830af0f21674fd28befee33cd7ca58bc0a68103829f2f9c" }
```

**Review by karthik2804 — APPROVED** [review]

**Comment by kate-goldenring** [comment]
Moving this back into draft because we will want to create a release and [update this link](https://github.com/fermyon/spin-kv-explorer/blob/1194a99f3bf8b56b270496e79450cf73c8e5891b/explorer/index.html#L262-L263) before we merge this template.

**Review by radu-matei — APPROVED** [review]

**Comment by radu-matei** [comment]
Feel free to merge this when ready.

**Pull request merged**
